ipwb/__main__.py

```diff
@@ -138,7 +138,7 @@ def checkArgs(argsIn):
     parser.add_argument(
         '-d', '--daemon',
         help='Location of ipfs daemon (default 127.0.0.1:5001)',
-        default='{0}:{1}'.format(IPFSAPI_HOST, IPFSAPI_PORT),
+        default=util.IPFSAPI_ADDRESS,
```

Comment:
I don't think we should rely on old host and port style. We can change this argument to accept multi-address which will simplify many things internally and will be more flexible to connect to IPFS nodes in many different ways.

Also, `util` is imported as `ipwbUtils` so we can make this one to be same as the rest.

Reply:
Made the `ipwbUtils` variable naming consistent in 8c28301.

Reply:
Specifying a multiaddress on the command line seems really awkward for a user, particularly with the variable rules based on whether it's an IP or a host. I'd rather the mutliaddress be inferred based on the IP and host that the user passes in.